assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -152,4 +152,18 @@ window.addEventListener( 'load', () => {
 			);
 		}
 	} );
+
+	const head = document.head || document.getElementsByTagName( 'head' )[ 0 ];
+	const style = document.createElement( 'style' );
+	const body = document.body || document.getElementsByTagName( 'body' )[ 0 ];
+	const backgroundColor = getComputedStyle( body ).backgroundColor;
+
+	head.appendChild( style );
+	style.appendChild(
+		document.createTextNode(
+			`:where(.wp-block-woocommerce-mini-cart-contents) {
+				background-color: ${ backgroundColor };
+			}`
+		)
+	);
```

Comment:
Nit: Things might have changed in browsers since I read about it, but I think if you do it the other way around (append the CSS into `style` and then `style` into `head`), the browser only needs to update the DOM once instead of twice.

```suggestion
	style.appendChild(
		document.createTextNode(
			`:where(.wp-block-woocommerce-mini-cart-contents) {
				background-color: ${ backgroundColor };
			}`
		)
	);
	head.appendChild( style );
```

Reply:
100% agree!